(docs): Fix vale warnings (#5225)
kedro-org/kedro: docs/integrations-and-plugins/pyspark_integration.md

        _spark_session.sparkContext.setLogLevel("WARN")
```

> You should modify this code to adapt it to your cluster's setup, for example by configuring the `spark.master` setting to `yarn` when you run Spark on [YARN](https://spark.apache.org/docs/latest/running-on-yarn.html).

Reviewer: ```suggestion
You should modify this code to match your cluster configuration, for example, by setting the master to `yarn` if your Spark cluster runs on [YARN](https://spark.apache.org/docs/latest/running-on-yarn.html).
```
